src/components/Icons/EnvoyFleet.tsx

```diff
@@ -0,0 +1,18 @@
+const EnvoyFleet: React.FC = () => (
```

Comment:
Just by looking at the name I don't understand what it is: a component or an icon. Maybe would be good to rename this component